lib/v4/MessageBuilder.ts

```diff
@@ -1,25 +1,20 @@
-import * as zod from 'zod/v4/core';
-import { type NonEmptyArray } from '../utils/NonEmptyArray.ts';
-import {
-  defaultErrorMap,
-  defaultErrorMapOptions,
-  isZodValidationErrorMap,
-} from './errorMap/index.ts';
+import { joinPath } from '../utils/joinPath.ts';
+import { isNonEmptyArray, type NonEmptyArray } from '../utils/NonEmptyArray.ts';
+import { titleCase } from '../utils/titleCase.ts';
+import type * as zod from 'zod/v4/core';
 
 export type ZodIssue = zod.$ZodIssue;
 
 export type MessageBuilder = (issues: NonEmptyArray<ZodIssue>) => string;
 
-const identityErrorMap: zod.$ZodErrorMap<zod.$ZodIssue> = (issue) => {
-  return issue.message;
-};
-
 export type MessageBuilderOptions = {
   prefix: string | null | undefined;
   prefixSeparator: string;
   maxIssuesInMessage: number;
   issueSeparator: string;
-  error: zod.$ZodErrorMap<zod.$ZodIssue> | false;
+  unionSeparator: string;
+  includePath: boolean;
+  forceTitleCase: boolean;
```

Comment:
dev: these options were previously passed to the `createErrorMap` function but I realized that they actually also apply to zod's error message and thus moved them to this module.